alphabase/spectral_library/flat.py

```diff
@@ -261,16 +262,16 @@ def get_full_charged_types(self, frag_df: pd.DataFrame) -> list:
         # Now if we have a fragment type that is a,b,c we should have the corresponding x,y,z
 
         corresponding = {"a": "x", "b": "y", "c": "z", "x": "a", "y": "b", "z": "c"}
-        loss_number_to_type = {0: "", 18: "_H2O", 17: "_NH3", 98: "_modloss"}
```

Comment:
`get_full_charged_types()` is deprecated, so there is no need to do any changes here.